src/components/Loader/index.tsx

```diff
@@ -5,12 +5,12 @@ type Props = CircularProgressProps & {
   inline?: boolean;
 };
 
-const GordonLoader = ({ size = 100, color = 'primary', inline = false }: Props) => {
-  if (inline) return <CircularProgress size={size} color={color} />;
+const GordonLoader = ({ size = 100, color = 'primary', inline = false }) => {
+  if (inline) return <CircularProgress size={size} color={color as Props['color']} />;
   return (
     <Grid className={styles.gordon_loader} container justifyContent="center" alignItems="center">
       <Grid item>
-        <CircularProgress size={size} color={color} />
+        <CircularProgress size={size} color={color as Props['color']} />
```

Comment:
This looks like a regression. We shouldn't need to override the type checking for `color`.